content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,114 @@
+---
+title: InfluxDB Cloud limits and adjustable quotas
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Adjustable quotas and limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account quotas per organization and hard global system limits.
```

Comment:
we use adjustable service quotas elsewhere... this says "account". 

Should we add that these are applied at a per organization level.  As we add the mutli-org support, this content would still hold true.

We could say something like.... currently adjustable service quotas apply on a per organization basis. Currently InfluxDB Cloud supports one organization per account.  

We then remove the 2nd sentence once multi-org lands?